ppr-ui/src/router/router.ts

```diff
@@ -28,4 +31,16 @@ router.afterEach((to, from) => {
   }
 })
 
+export function provideRouter() {
+  provide(RouterSymbol, router)
+}
+
+export function useRouter(): VueRouter {
+  const vueRouter: VueRouter = <VueRouter>inject(RouterSymbol)
+  if (!vueRouter) {
+    throw Error("Router cannot be injected, has not been provided");
+  }
+  return vueRouter
+}
```

Comment:
This pattern is borrowed from https://vue-composition-api-rfc.netlify.com/#plugin-development